pkg/sidecar_mounter/sidecar_mounter_config.go

```diff
@@ -276,6 +278,11 @@ func (mc *MountConfig) prepareConfigFile() error {
 			}
 		}
 	}
+	if mc.HostNetwork {
+		configMap["gcs-auth"] = map[string]interface{}{
+			"token-url": "unix:///gcsfuse-tmp/.volumes/gcs-fuse-csi-ephemeral/token.sock",
```

Comment:
Why not use a constant for this? Probably define it here, then it can be used in the main.go.